XML/TestCases/FunctionalTests-Network.xml

```diff
@@ -629,6 +629,31 @@
         <Tags>ethtool</Tags>
         <Priority>0</Priority>
     </test>
+    <test>
+        <testName>ETHTOOL-OFFLOADING-SETTING</testName>
+        <setupScript>.\Testscripts\Windows\SETUP-SR-IOV-Configure.ps1</setupScript>
+        <testScript>SRIOV-RUN-COMMON-TEST.ps1</testScript>
+        <setupType>TwoVM2Host1Dep</setupType>
+        <AdditionalHWConfig>
+            <Networking>SRIOV</Networking>
+        </AdditionalHWConfig>
+        <files>.\Testscripts\Linux\ETHTOOL-OFFLOADING-SETTING.sh,.\Testscripts\Linux\SR-IOV-Utils.sh,.\Testscripts\Linux\utils.sh</files>
+        <TestParameters>
+            <param>Remote_Script=ETHTOOL-OFFLOADING-SETTING.sh</param>
+            <param>Set_SSH=yes</param>
+            <param>NIC_COUNT=1</param>
+            <param>VF_IP1=10.0.0.40</param>
+            <param>VF_IP2=10.0.0.60</param>
+            <param>NETMASK=255.255.255.0</param>
+            <param>Switch_Name=SRIOV_MLNX</param>
+            <param>SECRET_PARAMS=(VM2Name,CheckpointName)</param>
+        </TestParameters>
+        <Platform>Azure,HyperV</Platform>
```

Comment:
@haiyangz Should it be a matter of specific platform for 'ethtool -K/k' command?

Reply:
The 'ethtool -K/k' command is common for Azure and Hyper-V based my test on them.